docs/parameters.md

```diff
@@ -0,0 +1,185 @@
+| `--database-ssl-mode` | SSL verification mode to use | `--foreman-db-sslmode` <br/> `--katello-candlepin-db-ssl-verify` <br/> `--katello-candlepin-db-ssl` <br/> `--foreman-proxy-content-pulpcore-postgresql-ssl`|
+| `--database-ssl-ca` | Path to the database CA certificate | `--foreman-db-root-cert` <br/> `--katello-candlepin-db-ssl-ca` <br/> `--foreman-proxy-content-pulpcore-db-ssl-root-ca` |
+| `--foreman-database-name` | Name of the Foreman database | `--foreman-db-database` |
+| `--foreman-database-user` | Owner of the Foreman database | `--foreman-db-username` |
+| `--foreman-database-password` | Password for Foreman database | `--foreman-db-password` |
+| `--foreman-database-pool` | Tunes the database pool for Foreman | `--foreman-db-pool` |
+| `--candlepin-database-name` | Name of the Candlepin database | `--katello-candlepin-db-name` |
+| `--candlepin-database-user` | Owner of the Candlepin database | `--katello-candlepin-db-user` |
+| `--candlepin-database-password` | Password for Candlepin database | `--katello-candlepin-db-password` |
+| `--pulp-database-name` | Name of the Pulp database | `--foreman-proxy-content-pulpcore-postgresql-db-name` |
+| `--pulp-database-user` | Owner of the Pulp database | `--foreman-proxy-content-pulpcore-postgresql-user` |
+| `--pulp-database-password` | Password for Pulp database | `--foreman-proxy-content-pulpcore-postgresql-password` |
+
+#### Certs
+
+##### Mapped
+
+| Parameter | Description | foreman-installer Parameter |
+| ----------| ----------- | --------------------------- |
+
+##### Unmapped
+
+| foreman-installer Parameter | Description | Reason |
+| --------------------------- | ----------- | ------ |
+
+#### Undetermined
+
+| foreman-installer Parameter | Description | Module | Puppet Parameter | Keep |
+| ------------------- | ----------- | ------ | ---------------- |-------------------|
+| `--foreman-foreman-service-puma-workers` | | foreman | foreman_service_puma_workers | `--foreman-puma-workers` |
+| `--foreman-foreman-service-puma-threads-min` | | foreman | foreman_service_puma_workers_min | `--foreman-puma-threads-min` |
+| `--foreman-foreman-service-puma-threads-max` | | foreman | foreman_service_puma_threads_max | `--foreman-puma-threads-max` |
+| `--foreman-dynflow-worker-instances` | | foreman | dynflow_worker_instances |
+| `--foreman-dynflow-worker-concurrency` | | foreman | dynflow_worker_concurrency |
+| `--foreman-loggers` | | foreman | loggers |
+| `--foreman-plugin-tasks-cron-line` | | foreman::plugin::tasks | cron_line |
+| `--foreman-plugin-tasks-automatic-cleanup` | | foreman::plugin::tasks | automatic_cleanup |
+| `--puppet-server` | | puppet | server |
+| `--puppet-server-ca` | | puppet | server_ca |
+| `--puppet-dns-alt-names` | | puppet | dns_alt_names |
+| `--puppet-ca-server` | | puppet | ca_server |
```

Comment:
I'd argue these should be in the "proxy" section, as they don't belong to the "main app"